src/ShareModal.scss

```diff
@@ -0,0 +1,56 @@
+@import '_variables';
+
+.ShareModal__content {
+    padding: 2rem;
+    border-bottom: 1px solid #E4E7EB;
+}
+
+.ShareModal__form {
+    display: flex;
```

Comment:
Need to update display flex to grid to apply grid template style.

Reply:
Done.